global.json

```diff
@@ -1,6 +1,6 @@
 {
   "sdk": {
-    "version": "9.0.101",
+    "version": "6.0.428",
```

Comment:
Do we need a version down to specific path here?
Or maybe we can use` "version": "6.0.4xx",` ? With this if you have 6.0.4xx it will run

Reply:
I don't know, I've decided to use the latest 6 version, because we are using the 6 version in the project. Looks like we can't use `x.x` based on the [documentation](https://learn.microsoft.com/en-us/dotnet/core/tools/global-json#version).

Reply:
Looks like it